### DISCUSSION TITLE
Coordinates (X and Y dimensions) problems running MintPy with GMTSAR

### DISCUSSION THREAD

**Discussion by anaferreira97** [insarlab/MintPy#760] [Q&A]
Hi!

I'm trying to run MintPy with GMTSAR but I'm getting this warning message:

![image](https://user-images.githubusercontent.com/98409351/162223288-c0856db4-3706-4e7c-b226-7aae05f41703.png)

I think that is a GDAL issue. I'm doing my "corr_ll.grd" and "unwrap_ll.grd" files, which is the projection from radar to lat and lon coordinates using the "proj_ra2ll.csh" and "trans.dat" files, but I'm getting the following coordinates:

![image](https://user-images.githubusercontent.com/98409351/162224037-67679ce0-d9b7-4fd5-af26-16d3bc22b11e.png)

I'm doing a case study in Coimbra, Portugal and the latitude looks good, but the longitude should be -8 or so. The DEM used was downloaded in topex. 

Could you help me find out if this is what is causing the warning problem?

**Comment by JPPereira93** [comment]
That is for sure a problem with the parameters you are using, check if the interferograms baselines are adequate for that case

**Comment by yunjunz** [comment]
Your 2nd screenshot looks reasonable to me. -8 and 352 are equivalent with 360 difference. I am not sure about the warning msg in your 1st screenshot: it's not from native mintpy function, my guess is that it's from gdal calls within prep_gmtsar.py. You could debug into this script to locate the source. Anyway, it's a warning, not error. So I would just proceed unless there is more severe issue rise up later.
